build.gradle

```diff
@@ -95,7 +95,7 @@ repositories {
     jcenter()
     maven {
         name = 'sponge'
-        url = 'http://repo.spongepowered.org/maven'
+        url = 'https://repo.spongepowered.org/maven'
```

Comment:
This should be changed to `https://repo.spongepowered.org/repository/maven-public/`, the new repository, the old url is not supported at all.

Reply:
Oh, yea, reading probably would've been helpful. I assume that then also the reference on line 19 needs to be changed as well?

Reply:
Yes, any and all references should be changed (and you should also look for other maven repositories using http and try to change them to https)

Reply:
Got it, thanks!